test/flycheck-test.el

```diff
@@ -50,12 +50,13 @@
 (require 'flycheck-ert)
 
 ;; Make a best effort to make Coq Mode available
-(mapc (lambda (dir)
-        (add-to-list 'load-path (expand-file-name "coq/" dir)))
-      '("/usr/share/emacs/site-lisp/"
-        "/usr/local/share/emacs/site-lisp/"))
+;; (mapc (lambda (dir)
+;;         (add-to-list 'load-path (expand-file-name "coq/" dir)))
+;;       '("/usr/share/emacs/site-lisp/"
+;;         "/usr/local/share/emacs/site-lisp/"))
+(add-to-list 'load-path "/usr/share/emacs/site-lisp/")
 
-(autoload 'coq-mode "coq")
+(autoload 'coq-mode "gallina")
```

Comment:
All the integration tests need the relevant mode to run the checker.  The Coq tests are unusual as they all start with:
```
  (skip-unless (load "coq" 'noerror 'nomessage))
```

With Coq installed, and using "gallina" instead, I can run the tests.  If I remove the explicit `load` calls or `autoload`, the tests are skipped.

So, I could put ProofGeneral in the VM, but since it's not available from MELPA (any reason why?), and cloning the Git from Cask doesn't work...  I'll have to follow the PG instructions to clone and `make`, which is a bit of a hassle.

But technically, we don't need PG to run the tests, only a mode called `coq-mode` :-)

I'm okay with reverting the `gallina` changes.